app/Http/Controllers/MeetingController.php

```diff
@@ -51,8 +51,10 @@ public function show($id)
 
     public function showDashboard()
     {
+        $meetings = Meeting::where('user_id', Auth::user()->id)->simplePaginate(16);
```

Comment:
Since the cards are sorted into 1, 2, 3 or 4 columns (depending on the screen size), maybe it'd be better to paginate with a number like 12, which divides from all of these numbers (regardless of screen size, it would always show an even amount of meetings in each row)

Reply:
That's actually smart af